elab_bridge/server_interface.py

```diff
@@ -14,7 +14,7 @@ def download_experiment(save_to, server_config_json, experiment_id, experiment_a
     save_to: str
         Path where to save the retrieved experiment data
     server_config_json: str
-        Path to the json file containing the redcap url, api token and required external modules
+        Path to the json file containing the eLabFTW url, api token and required external modules
```

Comment:
If you want to be more clear here and make it easy for the user to know what you talk about then you can list the keys that the json file needs to contain here instead of describing what you mean. The current keys are `api_token` and `api_url`. Also for elabftw, there are no external modules.